models/tridimensional/data_kleinstiver/cas9_mutants_stats.py

```diff
@@ -137,3 +137,106 @@ def main():
 
 if __name__ == '__main__':
```

Comment:
should push this to the bottom of the file and put any function declarations before it